.github/workflows/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: weekly
```

```suggestion
  - package-ecosystem: npm
    directory: /
    schedule:
      interval: weekly
      time: "23:00"
      timezone: Europe/London
    open-pull-requests-limit: 10
```
modify acording to your use case! Thanks to ping me

---

Thanks for the suggestion! I was checking [the docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit), because I never used `open-pull-requests-limit` before and I think that 10 is a good number for as. The default value is 5. 

In any case, we can always do changes later on 💯